patches/ui-webui-resources-tools-bundle_js_excludes.gni.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/ui/webui/resources/tools/bundle_js_excludes.gni b/ui/webui/resources/tools/bundle_js_excludes.gni
-index 544a35127374b54700255f5902e881fc5fb89d7d..31ceaa0ee30563ce5fb74bcad9dd39a8aece9507 100644
+index 544a35127374b54700255f5902e881fc5fb89d7d..404a2df5c69987461df2d18bd8383016b8cf6ee4 100644
 --- a/ui/webui/resources/tools/bundle_js_excludes.gni
 +++ b/ui/webui/resources/tools/bundle_js_excludes.gni
 @@ -14,6 +14,7 @@ foreach(excluded_file,
            "resources/js/load_time_data.js",
            "resources/ash/common/load_time_data.m.js",
            "resources/mwc/lit/index.js",
-+          "resources/brave/leo.bundle.js",
++          "resources/brave/leo.bundle.js", "resources/brave/opaque_ke.bundle.js",
```

Comment:
the lack of newline is intentional, right?

Reply:
Yup.

Reply:
this patch should be made extensible with a gn var. 
```
foreach(excluded_file,
brave_bundle_js_includes + 
      [ 
```